kolena/_utils/krequests.py

```diff
@@ -143,21 +144,22 @@ def delete(endpoint_path: str, api_version: int = DEFAULT_API_VERSION, **kwargs:
 def raise_for_status(response: requests.Response) -> None:
```

Comment:
Wondering if it is safer to just add a type ignore here instead of the changes to the errors for now. Want to limit the chances of breakages and these are widely used